can_tools/scrapers/official/SD/sd_vaccines.py

```diff
@@ -221,15 +191,30 @@ def normalize(self, resjson):
 
         # Dump records into a DataFrame and transform
         df = pd.DataFrame.from_records(data_rows)
-        # calculate metrics to match our def'ns
+
+        # Calculate metrics to match our definitions:
+        # SD moves individuals between buckets when they receive shots. E.g when someone gets their second dose of Moderna,
+        # they are removed from the 1-dose bucket and placed into the 2-dose bucket. So, we need to combine all the buckets.
         df["total_vaccine_completed"] = (
-            df["janssen_series"] + df["moderna_complete"] + df["pfizer_complete"]
+            df["janssen_series"]
+            + df["janssen_booster"]
+            + df["moderna_complete"]
+            + df["pfizer_complete"]
+            + df["moderna_booster"]
+            + df["pfizer_booster"]
         )
+
+        df["total_vaccine_initiated"] = (
+            df["moderna_1_dose"] + df["pfizer_1_dose"] + df["total_vaccine_completed"]
+        )
+
         df["total_vaccine_doses_administered"] = (
             df["janssen_series"]
             + df["moderna_1_dose"]
             + df["pfizer_1_dose"]
             + 2 * (df["moderna_complete"] + df["pfizer_complete"])
+            + 3 * (df["moderna_booster"] + df["pfizer_booster"])
+            + 2 * (df["janssen_booster"])
```

Comment:
FWIW- This is going to have some error in it because you can mix/match vaccines and boosters.

E.g. 1 janssen + 1 moderna booster (we'll count as 3 but should be 2) or 2 moderna + 1 janssen booster (we'll count as 2 but should be 3).

Might at least be worth a comment.  Or we could just drop `total_vaccine_doses_administered` since our pipeline doesn't need it.

Reply:
ah, good flag -- I'll just remove it!